aten/src/ATen/native/native_functions.yaml

```diff
@@ -13125,6 +13121,13 @@
   dispatch:
     CompositeExplicitAutograd: alias_copy_out
 
+- func: to_padded_tensor(Tensor self, float padding, int[]? output_size=None) -> Tensor
```

Comment:
@albanD

Reply:
I think it might not be torch scriptable if we add it on a class Tensor at the python level @cpuhrsch is this what you were getting at yesterday?

Reply:
Additionally I noticed that `to_dense` which seems like the analogous function to `to_padded_tensor` for sparse is a method registered as a native function

Reply:
If that's painful, you can keep it here for sure yes.